Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/CollectionViewGallery.cs

```diff
@@ -10,6 +10,7 @@ public CollectionViewGallery()
 			{
 				Children =
 				{
+					new Button { Text ="Enable CollectionView", AutomationId = "EnableCollectionView", Command = new Command(()=> Device.SetFlags(new[] { "CollectionView_Experimental" })) },
```

Comment:
Won't this blow away any flags which are already set? Will that be a problem if we want to test CV and Fast Renderers at the same time?

Reply:
That's a good question , need to try it. Do you had other idea to set this ? I also restart the gallery after the test finishes so any flags would be set again for next tests.

Reply:
maybe implement something like `Device.AddFlagIfNotExist` ? also remove...